ipinfo/cmd_download.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/ipinfo/cli/lib/complete"
+	"github.com/ipinfo/cli/lib/complete/predict"
+	"github.com/spf13/pflag"
+)
+
+const dbDownloadURL = "https://ipinfo.io/data/free/"
+
+var completionsDownload = &complete.Command{
+	Flags: map[string]complete.Predictor{
+		"--asn":         predict.Nothing,
+		"--country":     predict.Nothing,
+		"--country-asn": predict.Nothing,
+		"-f":            predict.Nothing,
+		"--format":      predict.Nothing,
+		"-t":            predict.Nothing,
+		"--token":       predict.Nothing,
+		"-h":            predict.Nothing,
+		"--help":        predict.Nothing,
+	},
+}
+
+func printHelpDownload() {
+	fmt.Printf(
+		`Usage: %s download [<opts>]
+
+Description:
+    Download the free ipinfo databases.
+
+Examples:
+    # Download country database in csv format.
+    $ %[1]s download --country --csv
+
+Options:
+  General:
+    --token <tok>, -t <tok>
+      use <tok> as API token.
+    --asn
+      download the free ipinfo asn database. 
+    --country
+      download the free ipinfo country database. 
+    --country-asn
+      download the free ipinfo country asn database. 
+    --format <fmt>, -f <fmt>
+      use the <fmt> output format of the database file.
+        mmdb (default.)
+        json
+        csv
```

Comment:
This isn't a good way to display the options one per line. Think of a nicer way. Hint: look at `mmdbctl import --help`.

Also not sure why this flag is under the `General` section when it's an output-related point. See `ipinfo bulk --help`.